report/unsafe/sections/albin_arrowhead_communication.tex

```diff
@@ -0,0 +1,92 @@
+project uses five of them nameley, service registry, orchestrator, authorization, gateway and gatekeeper. The service registry is used 
+to register systems and services, a system is defined as either a provider or a consumer. A provider provides a service for examample 
+the reading of a temperature sensor and in turn the consumer consumes that service which in this example might be to log the temperature
+in a database. A service is therefore what the provider is providing and the consumer consuming, more on this later. The orchestrator is
+used to decide which consumer and acces which provider and what services said provider is entitled to provide the conusmer with. The
+authorization is used to declare which providers are authozired within both the local cloud, intra cloud authorization, and between
+differnt clouds, inter cloud authorization. When defining inter or intra cloud rules to user also has to decide which interface should 
+handle the communication between the systems. These three systems are the minimun requirment for running a local arrowhead cloud. 
+To enable intra cloud communication both the gateway and gatekeeper core systems will have to be running as well. 
+
+\subsection{Communication}
+This project uses the arrowhead framework for Communication between the robot and the local arrowhead cloud as well as the 
+remote cloud from the factory. This section of the report aims to explain how the communication between these different clouds works as well
+as to define the purpose of using the arrowhead framework. This project decided to use arrowhead framework for communication between 
+the local cloud and the robot, all internal communication on the robot will be handled using ROS. This proides faster and more robust 
+control of the robot, not relying on internet connection and so forth to control the robot. It should be noted that the arrowhead framework
+is fully competent to control every aspect of the robot as well, but can run in to problems with delay in time critical sections.
+
+\subsubsection{Intra cloud communication}
+The prerequisites for using intra cloud communication is:
+\begin{itemize}
+    \item Running core services i.e. service registry, authorization and orchestrator.
+    \item A registered consumer and provider.
+    \item Defined intra cloud rules.
+\end{itemize}
+
+\subsubsection{Inter cloud communication}
+The prerequisites for using inter cloud communication is:
+\begin{itemize}
+    \item Running core services i.e. service registry, authorization, orchestrator, gateway and gatekeeper.
+    \item A running ActiveMQ relay server.
+    \item Defined inter cloud rules.
+\end{itemize}
+
+\subsubsection{Purpose of using arrowhead framework}
+The communication between different IoT-devices has been an issue for quite some time, and the security issues surrounding IoT-devices
```

Comment:
"The communication between different IoT-devices"
The communication insecurties between.. 
reason: might want to start sentence with whats the problem otherwise one might think there is some other issue regarding IoT communication.